backends/openvino/tests/ops/base_openvino_op_test.py

```diff
@@ -0,0 +1,167 @@
+import os
+import subprocess
+import tempfile
+import unittest
+
+import executorch
+
+import numpy as np
+import torch
+from executorch.backends.openvino.partitioner import OpenvinoPartitioner
+from executorch.backends.openvino.preprocess import OpenvinoBackend
+from executorch.exir import EdgeProgramManager, to_edge_transform_and_lower
+from executorch.exir.backend.backend_details import CompileSpec
+from torch.export import export, ExportedProgram
+
+
+class BaseOpenvinoOpTest(unittest.TestCase):
+    device = "CPU"
+    build_folder = ""
+
+    atol = 1e-1
+    rtol = 1e-1
```

Comment:
this seems like extremely low rtol/atol?

Reply:
Updated: https://github.com/ynimmaga/executorch/blob/db281b6b3b8aa504c620ffe36f603b097b74dd8b/backends/openvino/tests/ops/base_openvino_op_test.py#L19